pkg/ebpf/dns_snooper_test.go

```diff
@@ -4,6 +4,7 @@ package ebpf
 
 import (
 	bpflib "github.com/iovisor/gobpf/elf"
+	"k8s.io/apimachinery/pkg/util/rand"
```

Comment:
nit, did you mean to import `math/rand` here?

Reply:
🤦‍♂